seed/views/v3/users.py

```diff
@@ -0,0 +1,887 @@
+        if error:
+            return JsonResponse({
+                'status': 'error',
+                'message': message
+            }, status=status.HTTP_400_BAD_REQUEST)
+
+        ok, content = self.validate_request_user(pk, request)
+        if ok:
+            user = content
+        else:
+            return content
+
+        # If the only action requested is 'requires_superuser' no need to check an org affiliation
+        if len(actions) == 1 and actions[0] == 'requires_superuser':
+            return JsonResponse(
+                {'status': 'success', 'auth': {'requires_superuser': user.is_superuser}})
+
+        auth = self._try_parent_org_auth(user, org, actions)
+        if auth:
+            return JsonResponse({'status': 'success', 'auth': auth})
+
+        try:
+            ou = OrganizationUser.objects.get(user=user, organization=org)
+        except OrganizationUser.DoesNotExist:
+            return JsonResponse({'status': 'error', 'message': 'user does not exist'})
+
+        auth = {action: PERMS[action](ou) for action in actions}
+        return JsonResponse({'status': 'success', 'auth': auth})
+
+    def _parse_is_authenticated_params(self, request):
+        """checks if the org exists and if the actions are present
+
+        :param request: the request
+        :returns: tuple (actions, org, error, message)
+        """
+        error = False
+        message = ""
+        body = request.data
+        if not body.get('actions'):
+            message = 'no actions to check'
+            error = True
+
+        org_id = request.query_params.get('organization_id', None)
+        if org_id == '':
+            message = 'organization id is undefined'
+            error = True
+            org = None
+        else:
+            try:
+                org = Organization.objects.get(pk=org_id)
+            except Organization.DoesNotExist:
+                message = 'organization does not exist'
+                error = True
+                org = None
+
+        return body.get('actions'), org, error, message
+
+    def _try_parent_org_auth(self, user, organization, actions):
+        """checks the parent org for permissions, if the user is not the owner of
+        the parent org, then None is returned.
+
+        :param user: the request user
+        :param organization: org to check its parent
+        :param actions: list of str actions to check
+        :returns: a dict of action permission resolutions or None
+        """
+        try:
+            ou = OrganizationUser.objects.get(
+                user=user,
+                organization=organization.parent_org,
+                role_level=ROLE_OWNER
+            )
+        except OrganizationUser.DoesNotExist:
+            return None
+
+        return {
+            action: PERMS['requires_owner'](ou) for action in actions
+        }
+
+    @ajax_request_class
+    @action(detail=True, methods=['GET'])
+    def shared_buildings(self, request, pk=None):
```

Comment:
Was there anything that changed with this? I saw that we had a note down to investigate.

Reply:
In its current state, the GET successfully returns if the user has shared buildings enabled or not.  Did we want to change this?

Reply:
@nllong or @axelstudios This is the note we had `/api/v2/users/{id}/shared_buildings/ - get: investigate - bpd link`

Do either of you remember what the intent was behind that? (I'm not sure what bpd is.)

Reply:
Per discussions - this functionality hasn't been built out yet and copying over is enough for now.